app/src/components/MetricView.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { Alignment, Direction, Stack, Text } from '@kibalabs/ui-react';
+
+export interface MetricViewProps {
+  name: string;
+  value: number;
+}
+
+export const MetricView = (props: MetricViewProps): React.ReactElement => {
+  return (
+    <Stack directionResponsive={{ base: Direction.Horizontal, medium: Direction.Vertical }} isFullWidth={true} childAlignment={Alignment.Center} contentAlignment={Alignment.Center}>
```

Comment:
this stuff isnt needed here, this stack will always be vertical. sorry for the confusion.